src/riak_kv_pb_timeseries.erl

```diff
@@ -47,8 +47,9 @@
 -define(E_PUT,      4).
 -define(E_NOCREATE, 5).
 -define(E_NOT_TS_TYPE, 6).
--define(E_MISSING_TYPE, 7).
+-define(E_MISSING_TYPE,   7).
```

Comment:
Do we want extra spaces here (and for E_DELETE, below)? Not sure if you were trying to align the different macros in this block of code, but it doesn't quite look right here.